omod/src/main/java/org/openmrs/module/owa/web/controller/OwaRestController.java

```diff
@@ -47,18 +48,18 @@
 @Deprecated
 @Controller
 public class OwaRestController {
-	
+
```

Comment:
Looks Good to me but you need to address this indentation issues. This can also be seen in other areas of your code

Reply:
For any file that I work on, if it had white space my editor automatically removes it